ocs_ci/utility/deployment_openshift_logging.py

```diff
@@ -242,7 +242,7 @@ def create_instance_in_clusterlogging(sc_name=None):
     pod_obj = ocp.OCP(
         kind=constants.POD, namespace='openshift-logging'
     )
-    pod = pod_obj.wait_for_resource(
+    pod_status = pod_obj.wait_for_resource(
```

Comment:
you will also need to do assert on pod_status few lines below
Same for pvc_status